examples/OneOpenAir/OneOpenAir.ino

```diff
@@ -544,6 +569,7 @@ static void displayExecuteOta(OtaState state, String msg, int processing) {
         
         delay(1000);
       }
+      oledDisplay.setBrightness(0);
```

Comment:
Curious to know, what happens without this line?

Reply:
This like will be clear all display content while reboot device. If not the display still Show "Firmware update success" while rebooting